silence/silence_test.go

```diff
@@ -486,6 +486,8 @@ func TestSilenceLimits(t *testing.T) {
 	n, err := s.GC()
 	require.NoError(t, err)
 	require.Equal(t, 1, n)
+	// Need to remove the Id from the previous attempt.
+	sil2.Id = ""
```

Comment:
This is required because prior to this commit the maximum silences limit was checked before an `Id` was assigned to the silence. Now limits are checked afterwards.